index.js

```diff
@@ -2341,25 +2341,24 @@
   //. ```
   S.lastIndexOf = sanctifyIndexOf('lastIndexOf');
 
-  //# pluck :: Accessible a => TypeRep b -> String -> [a] -> [Maybe b]
+  //# pluck :: Accessible a => (b -> Boolean) -> String -> [a] -> [Maybe c]
```

Comment:
I'm not super happy with the type signatures. Both `b` and `c` are "unique type variables", so they may as well be `Any`. I wish there was some way to indicate that `c` is the set of values that pass the predicate inside the type signature. Perhaps a `Predicate a` pseudotype?